source/js/script/script.js

```diff
@@ -0,0 +1,31 @@
+var navMain = document.querySelector('.main-nav');
+var navToggle = document.querySelector('.main-nav__toggle');
+
+navToggle.addEventListener('click', function() {
+  if (navMain.classList.contains('main-nav--opened')) {
+    navMain.classList.remove('main-nav--opened');
+  } else {
+    navMain.classList.add('main-nav--opened');
+  }
+});
+
+function modalShow(className) {
+  var modal = document.querySelector('.modal');
+  var order = document.querySelectorAll(className);
```

Comment:
```suggestion
  const modal = document.querySelector('.modal');
  const orders = document.querySelectorAll(className);
```

когда all - множественное число